tasks/configuration.yml

```diff
@@ -0,0 +1,77 @@
+---
```

Comment:
Do we really need to notify handlers if we have `--config.reload.automatic` option in service unit? I don't know, need testing